__init__.py

```
@@ -160,6 +165,119 @@ def handle_fallback(self, message):
+    def parse_desktop_file(file_path: str, extra_langs: Optional[List[str]] = None) -> Dict[str, Union[str, List[str]]]:
+        """Parse a .desktop file to extract relevant application metadata.
+
+        Args:
+            file_path: Path to the .desktop file.
+            extra_langs: List of additional languages to consider.
+
+        Returns:
+            A dictionary containing the parsed application metadata.
+        """
+        extra_langs = extra_langs or []
+        extra_langs = [standardize_lang_tag(l) for l in extra_langs]
```

Ruff

Ambiguous variable name: `l`

(E741)